.changeset/stale-pianos-teach.md

```diff
@@ -0,0 +1,6 @@
+---
+"stylelint": minor
+---
+
+Fixed: `no-duplicate-selectors` end positions
+Fixed: `no-duplicate-selectors` false negatives for three or more duplicates
```

Comment:
[suggest] Just to be safe, can you separate the entries to each file? I don't remember if `changeset` supports multiple entries, and each entry should have different effects:

- "Fixed: `no-duplicate-selectors` end positions" -> patch
- "Fixed: `no-duplicate-selectors` false negatives for three or more duplicates" -> minor

Reply:
related: https://github.com/stylelint/stylelint/pull/7859#discussion_r1682913500

Reply:
I tried `npx changeset version` locally. Unfortunately, such double entries don't generate the expected content. Here's the diff in `CHANGELOG.md`:

```diff
diff --git a/CHANGELOG.md b/CHANGELOG.md
index f5bf24418..073dfd9b8 100644
--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,11 @@
 # Changelog
 
+## 16.8.0
+
+### Minor Changes
+
+- Fixed: `no-duplicate-selectors` end positions Fixed: `no-duplicate-selectors` false negatives for three or more duplicates ([#7867](https://github.com/stylelint/stylelint/pull/7867)) ([@romainmenke](https://github.com/romainmenke)).
+
 ## 16.7.0
 
 - Changed: `tap` formatter to support TAP14 ([#7759](https://github.com/stylelint/stylelint/pull/7759)) ([@Mouvedia](https://github.com/Mouvedia)).
```

So we have to separate entries for each file.